db/schema.rb

```diff
@@ -1496,7 +1496,6 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["program", "user_id"], name: "index_raffles_on_program_and_user_id", unique: true
-    t.index ["user_id"], name: "index_raffles_on_user_id"
```

Comment:
This is odd. I have this index locally and it's in production as well:

![CleanShot 2025-07-02 at 15 47 15@2x](https://github.com/user-attachments/assets/aa4a6d1a-0909-4bb5-a67d-dcf2d9a19c2c)

If we don't actually need it then we'll need a migration to drop it